src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1478,7 +1455,7 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 		}
 		//switch from long press selection to short press selection in pad selection mode
 		else if ((clip->lastSelectedParamID != kNoLastSelectedParamID) && padSelectionOn && multiPadPressSelected
```

Comment:
make sure a pad is not currently being held before switching to single press selection